apps/confessions-server/src/main.ts

```diff
@@ -0,0 +1,14 @@
+import * as dotenv from "dotenv";
+import * as path from "path";
+import { startApplication } from "./application";
+import { IS_PROD } from "./util/isProd";
+
+const dotEnvPath = IS_PROD
+  ? `/etc/secrets/.env`
```

Comment:
once you deploy this to render.com, be sure to make the deployment configuration create a `.env` file, and this directory is where it goes by default